src/components/elements/inputs/masked-input.js

```diff
@@ -64,3 +69,33 @@ export default ({ className, errors, onCommit, onUpdate, property, value }) => {
       value={value} />
   )
 }
+function createAlphabeticdMask(rawValue) {
```

Comment:
Thanks and it seem from googling that alphabetical and alphabetic are synonyms but alphabetical is more used so I'll change.